packages/server/modules/comments/graph/resolvers/comments.js

```diff
@@ -173,21 +203,32 @@ module.exports = {
 
     async commentEdit(parent, args, context) {
       // NOTE: This is NOT in use anywhere
-      await authorizeResolver(context.userId, args.input.streamId, 'stream:reviewer')
-      await editComment({ userId: context.userId, input: args.input })
-      return true
+      const stream = await authorizeStreamAccess({
+        streamId: args.input.streamId,
+        userId: context.userId,
+        serverRole: context.role,
+        auth: context.auth,
+        //public, but not public comments needs a stream role to do this
+        requireRole: true
+      })
+      const matchUser = !stream.role
+      try {
+        await editComment({ userId: context.userId, input: args.input, matchUser })
+        return true
```

Comment:
The function only has a return in one of its execution branches. You should either omit returns altogether (if it's a void return) or make sure all execution branches have one.

You could argue that it's impossible for anything other than `true` to be returned, but that would be more clear if the `return true` statement would be moved to the end of the function.